### ISSUE TITLE
Check NetSecConfig was able to load the policy do not happen after Android N

### ISSUE THREAD

**Issue by danielesegato** [datatheorem/TrustKit-Android#35]
I was looking at the library code and I noticed this check only run when the version is EXACTLY Android N. Shouldn't this be executed for later versions too? (`>=`)
```java
// On Android N, ensure that the system was also able to load the policy
if (Build.VERSION.SDK_INT == Build.VERSION_CODES.N) {
    // This will need to be updated/double-checked for subsequent versions of Android
    int systemConfigResId = getNetSecConfigResourceId(context);
    if (systemConfigResId == -1) {
        // Android did not find a policy because the supplied resource ID is wrong or the
        // policy file is not properly setup in the manifest, or contains bad data
        throw new ConfigurationException("TrustKit was initialized with a network policy " +
                "that was not properly configured for Android N - make sure it is in the " +
                "App's Manifest.");
    }
    else if (systemConfigResId != configurationResourceId) {
        throw new ConfigurationException("TrustKit was initialized with a different " +
                "network policy than the one configured in the App's manifest.");
    }
}
```

Am I missing something?

**Comment by nabla-c0d3** [comment]
Hello,
Thanks for the report - that's a good catch and we'll update this check.
Thanks!

**Comment by danielesegato** [comment]
Wanted to do a pull request but the project is too old and it needs many upgrades, libraries version etc.

And I see 3 test failing :) I think I'll leave this to you guys.

**Comment by jobot0** [comment]
@danielesegato It's in progress and should be released today or tomorrow :)

**Comment by jobot0** [comment]
Should be resolved with #36 (`1.1.0`)
